vnext/Shared/Threading/MessageQueueThreadFactory.cpp

```diff
@@ -8,7 +8,8 @@
 namespace Microsoft::ReactNative {
 
 std::shared_ptr<facebook::react::MessageQueueThread> MakeJSQueueThread() noexcept {
-  return std::make_shared<Mso::React::MessageDispatchQueue>(Mso::DispatchQueue::MakeLooperQueue(), nullptr, nullptr);
+  return std::make_shared<Mso::React::MessageDispatchQueue>(
+      Mso::DispatchQueue::MakeLooperQueue(nullptr), nullptr, nullptr);
```

Comment:
>nullptr [](http://example.com/codeflow?start=42&length=7)

It is difficult to read what this nullptr parameter is for. Should we rather use nullptr as a default parameter value, or have another override for the method with a different name for such case.

Reply:
I created another override for the method with different parameters, one taking no parameter and one taking the IReactNotificationService, please let me know your thoughts.